elastic/kibana
# [Discover] Save collapse sidebar state in local storage

## Summary

Fixes #8552

This PR adds persistence of sidebar callapse state. When refreshing page it will fit the last state from local storage.

### Test notes
- Open Discover, sidebar should be openned initially.
- Close sidebar
- Refresh the page

**Expected**: Sidebar should be closed. It should works visa-versa with openned sidebar.

**How it works after changes**

https://user-images.githubusercontent.com/39378793/135469210-ab651c81-86c5-494d-b38e-8c44304880d9.mp4
